doc/script_commands.txt

```diff
@@ -7278,8 +7283,8 @@ The function returns 1 if shop was updated successfully, or 0 if not found.
 
 NOTES:
  - That you cannot use -1 to specify default selling price!
```

Comment:
```suggestion
 - A sell price of -1 is not a valid value!
```

Reply:
Again - we could implement this.
As suggested by @Toshiro90